### ISSUE TITLE
Spanish translation for PACT Act updates from 10/26/22

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#48821]
## Issue Description

We made updates to the PACT Act R&S article about hazardous materials and presumptive conditions. We need to get these updates translated into Spanish.

Word doc sent to translators: [https://app.zenhub.com/files/133843125/e25c7935-419e-41a9-9a8a-e892b5c6f196/download](https://app.zenhub.com/files/133843125/e25c7935-419e-41a9-9a8a-e892b5c6f196/download)

## Tasks
- [x] Send text to translators (Randi)
- [x] Update Spanish page (Aliyah)
- [x] Review (Megan)
- [x] Publish (Megan)

## Acceptance Criteria
- [x] Spanish page has most recent updates

**Comment by RLHecht** [comment]
@aliyahblackmore here is the translation back from the State Department. Thanks![https://app.zenhub.com/files/133843125/d46b7671-84b3-447e-9840-d4209fc8d98a/download](https://app.zenhub.com/files/133843125/d46b7671-84b3-447e-9840-d4209fc8d98a/download)

**Comment by aliyahblackmore** [comment]
Hi @megzehn - I made the updates from this ticket and they are ready.
I'm moving to the other PACT Act updates now, so they can all be reviewed and published at the same time.

Preview link: http://preview-prod.vfs.va.gov/preview?nodeId=48330

**Notes:** 
1. Under “Con la Ley PACT se introducirán estos cambios:” the point that we needed to change had "20+" but was translated as “…más de 20.” Just noting here that the “+” sign was taken, likely because of translation differences. 
2. The links that are in this section should actually remain here as we moved them here due to them being over the character count for the Call to Action component.  I’ve moved these back on the Tagalog page. This is the preview link for the Tagalog page: http://preview-prod.vfs.va.gov/preview?nodeId=48613
